### PR TITLE
add online-console-extensions job

### PR DESCRIPTION
**Pull request by sallyom**
@stevekuznetsov please review, for now we'll have a simple script that checks 'grunt build', that's the only test in the online-console-extensions repo atm.

### CODE REVIEW

File: sjb/config/test_cases/test_branch_online_console_extensions.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  sync_repos:
+    - name: "online-console-extensions"
+  junit_analysis: False
+extensions:
+  actions:
+    - type: "script"
+      title: "online-console-extensions generated template check"
+      repository: "online-console-extensions"
+      script: |-
+        OS_BUILD_ENV_PULL_IMAGE=true OS_BUILD_ENV_PRESERVE=_output/scripts JUNIT_REPORT='true' ./hack/check-generated-templates.sh
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_branch_online_console_extensions.yml]: You probably don't need any of these env vars

Comment (sallyom) [reply sjb/config/test_cases/test_branch_online_console_extensions.yml]: you're probably right :) thanks

File: sjb/config/test_cases/test_branch_online_console_extensions.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  sync_repos:
+    - name: "online-console-extensions"
+  junit_analysis: False
+extensions:
+  actions:
+    - type: "script"
+      title: "online-console-extensions generated template check"
+      repository: "online-console-extensions"
+      script: |-
+        ./hack/check-generated-templates.sh
```
Comment (sallyom) [inline sjb/config/test_cases/test_branch_online_console_extensions.yml]: @stevekuznetsov that's all i need (until we add more checks/tests to the console-extensions repo)?

Comment (stevekuznetsov) [reply sjb/config/test_cases/test_branch_online_console_extensions.yml]: SGTM -- have you pushed the job out and tested it?

Comment (stevekuznetsov) [reply sjb/config/test_cases/test_branch_online_console_extensions.yml]: NVM, need the new AMI to do the clone correctly. duh.

**Pull request merged**